data/json/requirements/materials.json

```diff
@@ -86,5 +86,23 @@
     "type": "requirement",
     "//": "Materials for waterproofing leather and wooden items.",
     "components": [ [ [ "wax", 1 ], [ "pine_bough", 5 ] ] ]
+  },
+  {
+    "id": "fletching",
+    "type": "requirement",
+    "//": "Materials for fletching arrows.",
+    "components": [
+    [
```

Comment:
There should be an indentation of two spaces from the vertical line of `"components"`.

```
"components": [
  [
    [ "feather", 10 ],
    ...
  ]
]
```

Reply:
Fixed. Think that was causing it to fail?